### PR TITLE
TrkAna ini and cfg scripts

### PR DESCRIPTION
**Pull request by sophiemiddleton**
The .ini and .cfg scripts to run TrkAna on a given data set. Tested with MDC2020t musing already.

### CODE REVIEW

File: CampaignConfig/mdc2020_trkana.cfg [hunk 1]
```diff
@@ -0,0 +1,33 @@
+[global]
+includes = /mu2e/app/users/sophie/ProductionManager/Production/CampaignConfig/mdc2020_prolog.cfg
+release = MDC2020
+desc = MDC2020s
+release_v_i = r
+release_v_o = s
+db_version = v1_0
+db_purpose = perfect
+primary_name = IPAMuminusFlateMinus
```
Comment (brownd1978) [inline CampaignConfig/mdc2020_trkana.cfg:9]: The primary name is more specific than the name of this file would imply

File: CampaignConfig/mdc2020_trkana.cfg [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+trkana_version_counter = a
+trkana_version = v01_01_00
+beam = 1BB
```
Comment (brownd1978) [inline CampaignConfig/mdc2020_trkana.cfg:12]: Is this parameter needed for TrkAna?  I would think not.

File: CampaignConfig/mdc2020_trkana.cfg [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+email-to = sophie@fnal.gov
+[stage_trkana]
+job_setup.source_2 = /cvmfs/mu2e.opensciencegrid.org/Musings/TrkAna/%(trkana_version)s/setup.sh
+global.desc = %(release)s%(release_v_o)s_%(db_purpose)s_%(db_version)s_std_%(trkana_version)s
+global.stage_name = %(primary_name)sMix%(beam)s
+job_output_2.addoutput = *.%(stage_name)s.*.tka
+job_output_2.add_to_dataset = nts.%(submitter)s.%(stage_name)s.%(desc)s.tka
+job_output_2.dest = %(outdir_nts_tape)s/%(stage_name)s/%(desc)s/tka
+global.artRoot_dataset = nts.%(submitter)s.%(stage_name)s.%(desc)s.tka
+submit.n_files_per_job = 1
```
Comment (brownd1978) [inline CampaignConfig/mdc2020_trkana.cfg:22]: I think TrkAna can use a much larger merge factor than 1.  Please estimate a number based on an output file size of ~1GByte.

File: CampaignConfig/mdc2020_trkana.ini [hunk 4]
```diff
@@ -0,0 +1,42 @@
+[campaign]
```
Comment (brownd1978) [inline CampaignConfig/mdc2020_trkana.ini:1]: Where are memory and cpu/wall time limits set? TrkAna jobs should be able to run with very small footprints and time limits.

File: CampaignConfig/mdc2020_trkana.cfg (hunk 1 above)
Comment (sophiemiddleton) [reply CampaignConfig/mdc2020_trkana.cfg:9]: Yes, the primary can be edited to whatever the used wants, this was just an example

File: CampaignConfig/mdc2020_trkana.cfg (hunk 2 above)
Comment (sophiemiddleton) [reply CampaignConfig/mdc2020_trkana.cfg:12]: No, I can remove it

File: CampaignConfig/mdc2020_trkana.cfg (hunk 3 above)
Comment (sophiemiddleton) [reply CampaignConfig/mdc2020_trkana.cfg:22]: Will check

File: CampaignConfig/mdc2020_trkana.ini (hunk 4 above)
Comment (sophiemiddleton) [reply CampaignConfig/mdc2020_trkana.ini:1]: Will check

File: CampaignConfig/mdc2020_trkana.cfg (hunk 2 above)
Comment (sophiemiddleton) [reply CampaignConfig/mdc2020_trkana.cfg:12]: Hmm, I'm wondering. You might want to keep this, right, if you were doing studies.

**Comment by sophiemiddleton** [comment]
I guess I should close this PR and work on a TrkAna PR instead

**Pull request closed without merging**